src/cpp/core/FilePathTests.cpp

```diff
@@ -13,15 +13,18 @@
  *
  */
 
-// All of these tests presume Unix-style paths
-#ifndef _WIN32
-
 #define RSTUDIO_NO_TESTTHAT_ALIASES
 #include <tests/TestThat.hpp>
 
 #include <shared_core/Error.hpp>
 #include <shared_core/FilePath.hpp>
 
+#ifdef _WIN32
+# define kRootPrefix "C:/"
```

Comment:
Nit: these tests could still fail on anyone's local machine where the RStudio git drive isn't `C:`. This is really minor, but if there's a way to check the current drive letter, or maybe just compare against a regex (e.g. `[A-Z]`), or maybe by stripping `^[^:]:` from the start of the actual paths in the tests it could allow the tests to pass regardless of the drive name.

Reply:
Good idea -- perhaps I should just use the drive associated with the current working directory?

Reply:
I think that would be ideal, if that's possible.